distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -1,21 +1,47 @@
-import project.Data.Any
-import project.Data.Array
+import project.Data.Any.Any
+import project.Data.Array.Array
 import project.Data.Boolean
-import project.Data.Interval
+import project.Data.Vector.Vector
+import project.Nothing.Nothing
+
+export project.Data.Any.Any
+export project.Data.Array.Array
+export project.Data.Vector.Vector
+export project.Nothing.Nothing
+from project.Data.Boolean export all
+
+import project.Data.Filter_Condition.Filter_Condition
+import project.Data.Index_Sub_Range
+import project.Data.Interval.Bound
+import project.Data.Interval.Interval
+import project.Data.Maybe.Maybe
+import project.Data.Ordering.Ordering
+import project.Data.Text.Line_Ending_Style.Line_Ending_Style
+import project.Data.Text.Text_Sub_Range.Text_Sub_Range
+
+export project.Data.Filter_Condition.Filter_Condition
+export project.Data.Interval.Bound
+export project.Data.Interval.Interval
+export project.Data.Maybe.Maybe
+export project.Data.Ordering.Ordering
+export project.Data.Text.Line_Ending_Style.Line_Ending_Style
+export project.Data.Text.Text_Sub_Range.Text_Sub_Range
+
+from project.Data.Index_Sub_Range export Index_Sub_Range, First, Last
+
+
+
+
```

Comment:
Why is this space so big? I'm not against it but I'd just like to know where are our spacing rules coming from.

Reply:
As noted on PR am using this file to track what I have done. This just marks the divide.

Reply:
Ah okay, I did not realise that this whitespace is that mark. Maybe add a comment saying that? It would be more clear and also easier to find so win-win. Also more likely we will remember to get it sorted out at the end.